### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by Sapphireexe**
Finished Homework 6
This MR contains every task of HW_6 completed in three separated files:
hw_6_1.js  - 1 task completed.
hw_6_2.js - 4 tasks completed.
hw_6_3.js - 2 tasks completed.

hw_6_2 task 2 contains i = 0 on string 66. That's because I wasn't able to get better solution for an hour. As I know, there are two solutions in simple way - a few "if...else" conditions or a i = 0 counter instead of i-- or i -= 2.
This solution takes more time while running the code, but at least it works correct after my tests.

### CODE REVIEW

File: HW_5/hw_5_2.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+console.log(sizeConverter(1_000_000)); // kB
+console.log(sizeConverter(1_000_000_000_000_000)); // TB
+console.log(sizeConverter(1_099_511_627_776)); // TB
+console.log(sizeConverter(1_099_511_627_775)); // GB
+console.log(sizeConverter(1_048_576)); // MB
+console.log(sizeConverter(1_048_575)); // kB
+console.log(sizeConverter(1024)); // kB
+console.log(sizeConverter(1023)); // B
+
+function sizeConverter (a) {
```
Comment (anatoly-karpovich) [inline HW_5/hw_5_2.js:33]: Where is while loop?))

File: HW_6/hw_6_1.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    if (!employeeName || typeof employeeName !== 'string') {
+        throw new Error('Employee name "' + employeeName + '" is not a valid data type.');
+    };
+
+    for (let i = 0; i < employeeList.length; i++) {
+        if (employeeList[i] === employeeName) {
+            searchResult.push(employeeList[i], employeeSalary[i]);
+        };
+    };
```
Comment (anatoly-karpovich) [inline HW_6/hw_6_1.js:32]:
```suggestion
  const i = employeeList.indexOf(employeeName)
If(i == -1) return null
            searchResult.push(employeeList[i], employeeSalary[i]);
        };
    };
```

File: HW_6/hw_6_1.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    if (searchResult.length === 0) {
+        return null;
+    } else {
+        return searchResult;
+    };
+};
```
Comment (anatoly-karpovich) [inline HW_6/hw_6_1.js:39]:
```suggestion
    return searchResult;
};
```

File: HW_6/hw_6_2.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    if (!competitorPizzas.includes(myPizzas[i])) {
+      myUniquePizzas.push(myPizzas[i]);
+    };
+  };
+
+  if (myUniquePizzas.length !== 0) {
+    return myUniquePizzas;
+  } else if (myUniquePizzas.length === 0) {
+    return null;
+  };
```
Comment (anatoly-karpovich) [inline HW_6/hw_6_2.js:28]:
```suggestion
    return myUniquePizzas.length ? myUniquePizzas : null
  };
```

File: HW_6/hw_6_3.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  };
+
+  if (number < 10) {
+    return number;
+  } else {
+    return singleNumber (number);
+  };
+};
+
+console.log(singleNumber(19));
```
Comment (anatoly-karpovich) [inline HW_6/hw_6_3.js:26]: No handling for negative numbers. Try with -19

**Review by anatoly-karpovich — APPROVED** [review]

File: HW_6/hw_6_3.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+console.log('\n');
+
+/*
+6*. Написать функцию, которая принимает на вход строку с текстом, и заменяет каждую пару стоящих подряд идентичных букв на одну следующую в алфавите, 
+    и так пока в тексте не останется двух одинаковых букв стоящих рядом (через пробел и другой знак препинания можно)
+    Пример: aabc => bbc => cc => d
+*/
+
+function doubleLettersToNextLetter (inputString) {
```
Comment (VladimirLQA) [inline HW_6/hw_6_3.js:38]: No handling for `z` letter. Try this case `zzabc`

**Pull request merged**